src/app/components/ElectronicDelivery/ElectronicDeliveryForm.jsx

```diff
@@ -112,7 +112,7 @@ class ElectronicDeliveryForm extends React.Component {
                   }
                 </span>
               </div>
-              <span>You may request a maximum of 50 pages.</span><br />
+              <span>You may request a chapter, article or short excerpt.</span><br />
```

Comment:
Maybe a comma after `article`.

Reply:
The ticket's description doesn't have a comma there. I feel probably better following the description?

Reply:
Yay for Oxford commas!

Reply:
Yes, but I'd argue that we should have our content team (which I wish we still had) review any requested content changes.